data-processing/scripts/run_pipeline.py

```diff
@@ -335,8 +338,29 @@ def run_commands_for_arxiv_ids(
         type=str,
         help="Base url for definition model paths."
     )
+
+    parser.add_argument(
+        "--output-form",
+        type=str,
+        choices=[c.value for c in OutputForm],
```

Comment:
An alternative way of having users provide this argument could be as a variable-length argument with one word per format (e.g., `--output-form file db` instead of `both`). This might be helpful if we think we are going to add another type of output form in the future. Do you think that is likely?

Reply:
I think it's possible. And I think the alternative is cleaner too :) thanks for the suggestion! I've made the change.